client/me/two-step/index.jsx

```diff
@@ -137,12 +134,7 @@ module.exports = React.createClass( {
 		}
 
 		return (
-			<div>
-				<SectionHeader label={ this.translate( 'Backup Codes' ) } />
-				<Card className="me-security-backup-codes">
-					<Security2faBackupCodes userSettings={ this.props.userSettings } />
-				</Card>
-			</div>
```

Comment:
I moved this markup into the `Security2faBackupCodes` component that way I could more easily add/manipulate the `SectionHeader`.